### ISSUE TITLE
Worth maintaining?

### ISSUE THREAD

**Issue by victorb** [protocol/gaps#5]
GAPS application has not been working for quite a while. Google Apps does have a UI for discovering groups already. Currently it seems that we're paying Heroku around $32 a month for hosting the GAPS application and also engineering effort and time for maintaining it. In the end, is it worth it or should we just default to use the UI Google gives us instead?

**Comment by jesseclay** [comment]
tl;dr:  Personally I think fine to stop paying heroku for now. Google
interface is ok but not great, so we should figure out a replacement in the
not too distant future.

---

Current version of GAPS has significant security flaws, so decision was
made to take GAPS down until those are fixed. I'm not sure if Stripe is
using their [latest published version](https://github.com/stripe/gaps) or
not, but seems unlikely given last update of over a year ago.

Realistically I think GAPS will need a nearly complete re-write to be
useful for us. Though this will take time & energy, it may be worth it. To
the best of my knowledge the Google UI doesn't solve the following problems
that gaps was invented for:

- Easily see all mailing lists & those that are subscribed to,
subscribe/un-subscribe with one click
- Easily import filters into gmail so groups are sorted automatically
(though gmail oauth & automatic filter importing has always been broken in
GAPS, so need to download all the filters & import manually)

At the least we can do the above things in the google ui for now, but if
we're going to keep using mailing lists (v. useful imo) then I think we
should put the GAPS re-write in the pipeline (or maybe there are other
solutions already released? I haven't looked deeply). This seems like a
project that would be especially well suited for someone in PL Team or a
vendor.

**Comment by dignifiedquire** [comment]
I would like to enable it again, as it is really useful to see all existing lists, even when it is not super great in terms of usability.
